app/javascript/packs/application-common.js

```diff
@@ -64,3 +65,5 @@ miqOptimizationInit();
 
 import * as move from '../helpers/move.js';
 ManageIQ.move = move;
+
+ManageIQ.redux.store.dispatch(initNotifications(true));
```

Comment:
This is causing an API request on the login screen, causing a browser login popup.